x/lockup/client/cli/tx.go

```diff
@@ -62,3 +63,20 @@ func NewForceUnlockByIdCmd() (*osmocli.TxCliDesc, *types.MsgForceUnlock) {
 		Flags: osmocli.FlagDesc{OptionalFlags: []*pflag.FlagSet{FlagSetUnlockTokens()}},
 	}, &types.MsgForceUnlock{}
 }
+
+// NewRebondTokensCmd allows a user to rebond their tokens if they have unlocked them.
+func NewRebondTokensCmd() (*osmocli.TxCliDesc, *types.MsgRebondTokens) {
+	return &osmocli.TxCliDesc{
+		Use:   "rebond-tokens [id]",
+		Short: "rebond unlocking tokens",
+		Long: `Rebond unlocking tokens. If the amount flag is not specified, will rebond all tokens
+
+Example:
+osmosisd tx lockup rebond-tokens 1 
+		`,
+		CustomFlagOverrides: map[string]string{
+			"coins": FlagAmount,
```

Comment:
`CustomFlagOverrides` in this case means that we will be providing `Coins` field in `MsgRebondTokens` via flag `--FlagAmount` and not `--coins`

So right now it will look like this: `--amount X`, so I think we should keep it like it is now, or wdyt about adding a new flag for each 'type' of amount, be it a rebonding amount or unlocking amount?

Reply:
it is not possible to do it here, since "coins" here corresponds to an existing field in a `MsgRebondTokens` struct, there is no `rebond-amount` field in the struct 

`CustomFlagOverrides` essentially takes a field and makes it so, that the value to this field is parsed via flag, not argument